### PR TITLE
Delete metrics after flush

### PR DESCRIPTION
**Pull request by AdamMagaluk**

### CODE REVIEW

File: config.js [hunk 1]
```diff
@@ -58,7 +58,7 @@ Optional Variables:
   deleteGauges:     don't send values to graphite for inactive gauges, as opposed to sending the previous value [default: false]
   deleteTimers:     don't send values to graphite for inactive timers, as opposed to sending 0 [default: false]
   deleteSets:       don't send values to graphite for inactive sets, as opposed to sending 0 [default: false]
-  deleteCounters:   don't send values to graphite for inactive counters, as opposed to sending 0 [default: false]
+  deleteCounters:   don't send values to graphite for inactive counte rs, as opposed to sending 0 [default: false]
```
Comment (kevinswiber) [inline config.js:61]: WTF?

Comment (mdobson) [inline config.js:61]: Too excited fixing a bug

**Comment by mdobson** [comment]
+1

File: config.js (hunk 1 above)
Comment (AdamMagaluk) [inline config.js:61]: haha emacs slipped

**Pull request merged**